xml/System.Numerics/BigInteger.xml

```diff
@@ -2728,10 +2728,26 @@ value Mod 2 = 0
         <Parameter Name="value" Type="System.Decimal" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The value to convert to a <see cref="T:System.Numerics.BigInteger" />.</param>
+        <summary>Defines an explicit conversion of a <see cref="T:System.Decimal" /> object to a <see cref="T:System.Numerics.BigInteger" /> value.</summary>
+        <returns>An object that contains the value of the <paramref name="value" /> parameter.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ Any fractional part of the `value` parameter is truncated before conversion.
+
+ The overloads of the Explicit(Decimal to BigInteger) method define the types to which or from which a <xref:System.Numerics.BigInteger> object can be converted. Because the conversion from <xref:System.Decimal> to <xref:System.Numerics.BigInteger> can involve truncating any fractional part of `value`, language compilers do not perform this conversion automatically. Instead, they perform the conversion only if a casting operator (in C#) or a conversion function (such as `CType` in Visual Basic) is used. Otherwise, they display a compiler error.
```

Comment:
Explicit(Decimal to BigInteger) should be `<xref:System.Numbers.BigInteger.op_Explicit%2A>`. That's probably true of this paragraph in each version of the overloads.